similar/editor/med.cpp

```diff
@@ -319,38 +319,62 @@ static int padnum=0;
 static void init_editor_screen();
 static void gamestate_restore_check();
 static window_event_result editor_handler(UI_DIALOG *dlg,const d_event &event, unused_ui_userdata_t *data);
+static void close_editor();
 
 namespace dsx {
 void init_editor()
 {
+	const char *const pads[] = {
```

Comment:
Can this be made `static`?  Given the consistency of size, you might get better packing using `static const char pads[][13]` (or `[16]` for quick indexing) than using pointers to anonymous strings.  On x64 systems, the pointers will cost 8 bytes each, so if you waste an average of fewer than 8 bytes per string, storing it in a single contiguous array should be smaller.